src/Cardiompp.Application/Services/AuthenticationService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Cardiompp.Application.DataContracts.v1.Requests.Login;
+using Cardiompp.Application.Services.Contracts;
+using Cardiompp.Domain.Repositories;
+using System.Threading.Tasks;
+
+namespace Cardiompp.Application.Services
+{
+    public class AuthenticationService : IAuthenticationService
+    {
+        private readonly IUnitOfWork _unitOfWork;
+
+        public AuthenticationService(IUnitOfWork unitOfWork)
+        {
+            _unitOfWork = unitOfWork;
+        }
+
+        public async Task<bool> AuthenticateAsync(string name, string password)
+        {
+            var isAuthenticate = false;
+            var cardiomppAgent = await _unitOfWork.CardiomppAgentRepository.GetByNameAndPassword(name, password);
+
+            if (cardiomppAgent != null)
+                isAuthenticate = true;
+
+            return isAuthenticate;
```

Comment:
Esse método pode ser simplificado
```suggestion
            var cardiomppAgent = await _unitOfWork.CardiomppAgentRepository.GetByNameAndPassword(name, password);

            return cardiomppAgent != null;
```

Reply:
Feito. 

![image](https://user-images.githubusercontent.com/12301754/80872630-789a2e80-8c89-11ea-9a37-23163591977c.png)